kolibri/core/assets/src/constants.js

```diff
@@ -114,6 +114,18 @@ export const SyncStatus = {
   NOT_CONNECTED: 'NOT_CONNECTED',
 };
 
+export const DownloadRequestStatus = {
+  QUEUED: 'QUEUED',
+  DOWNLOADED: 'DOWNLOADED',
+  FAILED: 'FAILED',
+};
+
+export const DownloadRequestReason = {
+  USER_INITIATED: 'USER_INITIATED',
+  DOWNLOADED_BY_ADMIN: 'DOWNLOADED_BY_ADMIN',
```

Comment:
I had suggested that we annotated this directly onto the ContentNode object itself, as I think adding a separate DownloadRequest object for every ContentNode imported by an admin could be onerous, so I think we can probably skip this constant.